include/uid2/timestamp.h

```diff
@@ -18,7 +18,7 @@ namespace uid2
 		std::int64_t GetEpochMilli() const { return EpochMilli; }
 		bool IsZero() const { return EpochMilli == 0; }
 
-		Timestamp AddSeconds(int seconds) const { return Timestamp(EpochMilli + seconds * 1000); }
+		Timestamp AddSeconds(std::int64_t seconds) const { return Timestamp(EpochMilli + seconds * 1000); }
```

Comment:
curious, why this change?

Reply:
It was overflowing when adding the token expiry